internal/util/parser.go

```diff
@@ -0,0 +1,43 @@
+package util
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"regexp"
+	"strings"
+)
+
+func GetDescriptionFromFunction(absPath string, re *regexp.Regexp) (string, error) {
+	if !IsFile(absPath) {
+		return "", fmt.Errorf("Unable to find file path to function. Expected path: '%s'.", absPath)
+	}
+
+	fset := token.NewFileSet()
```

Comment:
I don't like the name "fset".
In general use more descriptive names.
We don't need to go overboard, but we also don't need to worry about the size of our source files.